src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsSinkTask.java

```diff
@@ -91,4 +98,14 @@ public void stop() {
     public String version() {
         return getClass().getPackage().getImplementationVersion();
     }
+
+    protected long computeRetryWaitMs(int retryAttempts, long retryBackoffMs) {
+        if (retryAttempts > 0 && retryAttempts <= 32) {
+            long waitDuration = retryBackoffMs << retryAttempts;
+            waitDuration = Math.min(waitDuration, MAX_RETRY_TIME_MS);
+            return ThreadLocalRandom.current().nextLong(0, waitDuration);
```

Comment:
I do not remember whether `computeRetryWaitMs` must be thread safe or no. It not, using `ThreadLocalRandom` is not required.

Reply:
I think it might, that's why confluent also uses `ThreadLocalRandom` in their implementation.